public/app/features/alerting/unified/mocks/server/configure.ts

```diff
@@ -140,6 +124,15 @@ export const disablePlugin = (pluginId: SupportedPlugin) => {
 };
 
 /** Make alertmanager config update fail */
-export const makeGrafanaAlertmanagerConfigUpdateFail = () => {
-  server.use(updateGrafanaAlertmanagerConfigHandler(ALERTMANAGER_UPDATE_ERROR_RESPONSE));
+export const makeAlertmanagerConfigUpdateFail = () => {
+  server.use(updateAlertmanagerConfigHandler(ALERTMANAGER_UPDATE_ERROR_RESPONSE));
```

Comment:
why we don't use a message in parameter as we do in the `makeAllAlertmanagerConfigFetchFail`?

Reply:
Fair point, I was being lazy 😁 I will add this to this method as well so we have the same capability